src/main/java/Factorial/Factorial.java

```diff
@@ -0,0 +1,16 @@
+package Factorial;
+
+public class Factorial {
+    public static void main(String[] args) {
+        int number = 4;
+        System.out.println(findFactorial(number));
+    }
+
+    public static int findFactorial(int number) {
+        int i,factorialResult=1;

```

Comment:
```suggestion
        int i,factorialResult = 1;
```
operasyon ile field arasina bir bosluk birakir misin